client/python/README.md

```diff
@@ -1,31 +1,39 @@
-# Armada Python client
+Armada Python Client
+=
+<hr />
 
+## How To Use This Application
+***
+
+> **These instructions are intended for end-users.** If you wish to develop against armada_client, please see our **[documentation on contributing](CONTRIBUTING.md )**
+
+1) Click on [armada_client](https://pypi.org/project/armada-client/).
+2) Read the file [CONTRIBUTING.md](CONTRIBUTING.md)
+3) Click [here](https://armadaproject.io/libraries) get an overview of Armada client libraries
 
 Python client wrapping the gRPC services defined in `submit.proto` and `events.proto`; allows for
 
-- submitting, cancelling, and reprioritising jobs, and
+- submitting, cancelling, and re-prioritising jobs, and
 - watching for job events.
 
 
 ## Build
 Prerequisites:
 
 For building the python client:
-In the repo level, run `make python`
-`cd client/python`
-`poetry install`
+In the repo level, run `make python` then `cd client/python` and lastly run `pip install` to install the client from git with all necessary dependencies.
```

Comment:
```suggestion
In the repo level, run `make python` then `cd client/python` and lastly run `pip install .` to install the client from git with all necessary dependencies.
```

Reply:
@kindyluv if this comment is addressed (you can just hit the commit suggestion button) this PR will become mergeable